libs/utils/trace_analysis.py

```diff
@@ -78,6 +78,31 @@ def setXTimeRange(self, t_min=None, t_max=None):
         logging.info('Set plots time range to (%.6f, %.6f)[s]',
                 self.x_min, self.x_max)
 
+    def plotFunctionStats(self, functions=None, metrics='avg'):
+        if not hasattr(self.trace, '_functions_stats_df'):
+            logging.warning('Functions stats data not available')
+            return None
+        df = self.trace.functions_stats_df(functions)
+        if not isinstance(metrics, list):
+            metrics = [metrics]
+        for _m in metrics:
+            if _m.lower() not in df.columns:
+                msg = 'Metric [{}] not supported, available metrics are {}'\
+                        .format(_m, df.columns.tolist())
+                raise ValueError(msg)
```

Comment:
Right... probably better... to avoid a moon travel in case we have just a GP since the beginning...